docs/source/manual/core.rst

```diff
@@ -627,12 +627,30 @@ You can then add this task to your application's environment:
     environment.admin().addTask(new TruncateDatabaseTask(database));
 
 Running a task can be done by sending a ``POST`` request to ``/tasks/{task-name}`` on the admin
-port. For example::
+port. The task will receive any query parameters as arguments. For example::
 
     $ curl -X POST http://dw.example.com:8081/tasks/gc
     Running GC...
     Done!
 
+You can also extend ``PostBodyTask`` to create a task which uses the body of the post request. Here's an example:
+
+.. code-block:: java
+
+    public class EchoTask extends PostBodyTask {
+        private final Database database;
```

Comment:
`database` field isn't initialized or used in this example, so please delete this line.